crates/core/tedge_api/src/workflow/state.rs

```diff
@@ -90,34 +164,42 @@ impl GenericCommandState {
 
     /// Update the command state with a new status describing the next state
     pub fn move_to(mut self, status: String) -> Self {
-        GenericCommandState::inject_text_property(&mut self.payload, "status", &status);
+        GenericCommandState::inject_text_property(&mut self.payload, STATUS, &status);
 
         GenericCommandState { status, ..self }
     }
 
     /// Update the command state to failed status with the given reason
     pub fn fail_with(mut self, reason: String) -> Self {
-        let status = "failed";
-        GenericCommandState::inject_text_property(&mut self.payload, "status", status);
-        GenericCommandState::inject_text_property(&mut self.payload, "reason", &reason);
+        let status = FAILED;
+        GenericCommandState::inject_text_property(&mut self.payload, STATUS, status);
+        GenericCommandState::inject_text_property(&mut self.payload, REASON, &reason);
 
         GenericCommandState {
             status: status.to_owned(),
             ..self
         }
     }
 
+    /// Mark the command as terminated
+    pub fn terminate(self) -> Self {
```

Comment:
Optional:
```suggestion
    /// Clear the command as complete
    pub fn clear(self) -> Self {
```
Since `terminate` has a negative connotation to it, implying a failed path.

Reply:
renamed by https://github.com/thin-edge/thin-edge.io/pull/2792/commits/c560c87d4213dee7ca4d505481ae032c6e35f026